Course_Part_04/src/Assignment12_Methods.java

```diff
@@ -0,0 +1,47 @@
+public class Assignment12_Methods {
+
+    public static void main(String[] args) {
+        printGreetingMessageToConsole();
+        System.out.println(getJavaGreeting());
+        printNameToConsole("Hello John, welcome to the Java course :)");
```

Comment:
De bedoeling is om alleen de naam mee te geven als parameter en de rest van de tekst in de methode op te nemen, desalniettemin prima gedaan.